config/fcos/v1_6_exp/validate.go

```diff
@@ -22,6 +22,23 @@ import (
 	"github.com/coreos/vcontext/report"
 )
 
+const rootDevice = "/dev/disk/by-id/coreos-boot-disk"
+
+// Validate() operates on the top-level Config rather than Disk because
+// it needs to check all partitions in all disks, not just one disk
```

Comment:
What do you think of 
"// Validate() operates on the top-level Config because the Disk struct is defined in base.Config"?

Reply:
That's a good start but I'd probably be a bit more explicit.  Maybe e.g. "We can't define a Validate function directly on Disk because that's defined in base, so we use a Validate function on the top-level Config instead."